src/main/java/pl/simpleascoding/tutoringplatform/advertisement/AdvertisementModelMapper.java

```diff
@@ -0,0 +1,13 @@
+package pl.simpleascoding.tutoringplatform.advertisement;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import pl.simpleascoding.tutoringplatform.advertisement.dto.Advertisement;
+
+@Mapper(componentModel = "spring")
+interface AdvertisementModelMapper {
```

Comment:
we should decide what we want to sign as public and what we don't

Reply:
> 

'we should decide what we want to sign as public and what not"